receivers/util.go

```diff
@@ -111,6 +115,45 @@ func JoinURLPath(base, additionalPath string, logger logging.Logger) string {
 	return u.String()
 }
 
+// LOGZ.IO GRAFANA CHANGE :: DEV-43657 - Set logzio APP URLs for the URLs inside alert notifications
+
+func ToLogzioAppPath(path string) string {
+	// because the app path contains # symbol which is encoded by default we need to replace encoded symbol back to original
+	return strings.Replace(path, EncodedHashSymbol, "#", 1)
+}
+
+func AppendSwitchToAccountQueryParam(u *url.URL, accountId string) *url.URL {
+	const LogzioSwitchToAccountQueryParamName = "switchToAccountId"
```

Comment:
(not critical) i assume it's from the copied code - but why do we need it as const if it's inside function scope? 🤔 
also starting in cap means it is public, which also doesn't make much sense if it's in scope of function ? did the IDE not give any warning on this? 

i am not sure because it might be a golang thing 😅

Reply:
it's const as it doesn't change. changed the capital to small.